src/Business Foundation/Test/NoSeries/src/NoSeriesTests.Codeunit.al

```diff
@@ -957,6 +958,58 @@ codeunit 134530 "No. Series Tests"
         Assert.AreEqual('', NoSeries.GetLastNoUsed(''), 'GetLastNoUsed should return empty code if argument supplied is empty code');
     end;
 
+    [HandlerFunctions('NoSeriesLineDrilldownHandler')]
+    [Test]
+    procedure TestCreateNewNoSeriesFromUI()
+    var
+        DummyNoSeriesRec: Record "No. Series";
+        NoSeries: TestPage "No. Series";
+        NewNoSeriesCode: Code[20];
+    begin
+        // Setup
+        NewNoSeriesCode := CopyStr(Any.AlphabeticText(MaxStrLen(DummyNoSeriesRec.Code)), 1, MaxStrLen(DummyNoSeriesRec.Code));
+
+        // Execute
+        NoSeries.OpenNew();
+        NoSeries.Code.SetValue(NewNoSeriesCode);
+        NoSeries.Next();
+        NoSeries.GoToKey(NewNoSeriesCode);
+        NoSeries.StartNo.Drilldown();
+
+        // Verify
+        NoSeries.GoToKey(NewNoSeriesCode);
+        LibraryAssert.AreEqual('00001', NoSeries.StartNo.Value(), 'Starting No. was not as expected');
+    end;
+
+    [Test]
+    procedure TestNoSeriesCodeFilterChange()
+    var
+        NoSeriesTests: Codeunit "No. Series Tests";
+        NoSeries: TestPage "No. Series";
+    begin
+        // Setup
+        BindSubscription(NoSeriesTests);
```

Comment:
Do you want to unbind as well at the end?

Reply:
no